app/boards/arm/corneish_zen/corneish_zen_v2_left.dts

```diff
@@ -0,0 +1,81 @@
+/*
+*
+* Copyright (c) 2021 Darryl deHaan
+* SPDX-License-Identifier: MIT
+*
+*/
+
+#include "corneish_zen.dtsi"
+
+/{
+	chosen {
+		zephyr,display = &epd;
+	};
+
+	kscan0: kscan {
+		compatible = "zmk,kscan-gpio-matrix";
+		label = "KSCAN";
+
+		diode-direction = "col2row";
+		row-gpios
+			= <&gpio0 19 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&gpio0 4 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&gpio0 31 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&gpio0 30 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			;
+
+		col-gpios
+			= <&gpio0 21 GPIO_ACTIVE_HIGH>
+			, <&gpio0 23 GPIO_ACTIVE_HIGH>
+			, <&gpio0 12 GPIO_ACTIVE_HIGH>
+			, <&gpio1  9 GPIO_ACTIVE_HIGH>
+			, <&gpio0  7 GPIO_ACTIVE_HIGH>
+			, <&gpio0  5 GPIO_ACTIVE_HIGH>
+			;	
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		blue_led: led_0 {
+			gpios = <&gpio0 16 GPIO_ACTIVE_HIGH>;
+			label = "Blue LED";
+		};
+	};
+
+    chosen {
+		zmk,battery = &vbatt;
+	};
+
+    vbatt: vbatt {
```

Comment:
```suggestion
	vbatt: vbatt {
```

Reply:
This one is strange.... it looks fine VS Code locally, but when I push it there is misalignment